config_spec.yaml

```diff
@@ -23,6 +23,41 @@ accept-keysend:
   description: |
     Allow others to send payments directly to your public key through keysend instead of having to get a new invoice
   default: false
+accept-amp:
+  type: boolean
+  name: Accept Spontaneous AMPs
+  description: |
+    If true, spontaneous payments through AMP will be accepted. Payments to AMP
```

Comment:
Rather than "if true", I think "if on" sounds more broadly appropriate. There are several instances in the config where this is the case.